### DISCUSSION TITLE
Paramétre CDC influence ambiance

### DISCUSSION THREAD

**Discussion by DavMar05** [witsa/synapps#521] [Ideas]
une idée cdc avec influence ambiance

Lors de l'utilisation et de l'exploitation du reflet courbe de chauffe avec influence ambiance sur studio Je me suis rendu compte qu'il manquait des paramètres importants. Ils seraient sympas d'avoir les paramètres de hors-gel ou juste le hors gel. ( C'est une idée car on peut aussi  bidouiller)

**Comment by WITNicolas** [comment]
Pour appuyer la demande, l'idée serait que les REFLETS puisse compenser un peu les manques qu'il peut y avoir dans les ressources au niveau de l'exploitation.
En effet certaines propriétés (pourtant trés utile à l'Exploitation) ne sont pas accessibles justement à un niveau EXPLOITANT.
Il ne faut pas qu'il y ait non plus tout dans les Reflets :) :)
